test/it/pipeline/src/test/java/org/apache/shardingsphere/test/it/data/pipeline/core/task/InventoryTaskTest.java

```diff
@@ -77,7 +76,7 @@ public void assertStartWithGetEstimatedRowsFailure() throws ExecutionException,
         PipelineTableMetaDataLoader metaDataLoader = new StandardPipelineTableMetaDataLoader(dataSource);
         try (
                 InventoryTask inventoryTask = new InventoryTask(inventoryDumperConfig, taskConfig.getImporterConfig(),
-                        PipelineContextUtil.getPipelineChannelCreator(), new DataSourceImporterConnector(DATA_SOURCE_MANAGER), dataSource,
```

Comment:
TODO, In concurrent scenarios, importer may not stop causing ci timeout, I will follow up on this